inc/TRestGeant4ToDetectorHitsProcess.h

```diff
@@ -69,15 +70,18 @@ class TRestGeant4ToDetectorHitsProcess : public TRestEventProcess {
     void PrintMetadata() override;
 
     /// Returns the name of this process
-    const char* GetProcessName() const override { return "g4toHitsEvent"; }
+    const char* GetProcessName() const override { return "geant4toHits"; }
 
     // Constructor
     TRestGeant4ToDetectorHitsProcess();
-    TRestGeant4ToDetectorHitsProcess(const char* configFilename);
+
+    explicit TRestGeant4ToDetectorHitsProcess(const char* configFilename);
+
     // Destructor
-    ~TRestGeant4ToDetectorHitsProcess();
+    ~TRestGeant4ToDetectorHitsProcess() override;
 
-    ClassDefOverride(TRestGeant4ToDetectorHitsProcess, 1);  // Transform a TRestGeant4Event event to a
+    ClassDefOverride(TRestGeant4ToDetectorHitsProcess, 2);  // Transform a TRestGeant4Event event to a
```

Comment:
I think this is not necessary since only non-stream data members have changed name